invenio_previewer/templates/semantic-ui/invenio_previewer/bottom.html

```diff
@@ -61,18 +48,25 @@ <h3 id="embedModalLabel">{{_('Embed code')}}</h3>
 
 
 {# Add div to push down bottom if necessary #}
-<script>
-  $('#wrap').append('<div id="push"></div>');
+<script type="module">
+  $('.ui.button.embed-btn').click(function () {
+    $('#embedModal').modal('show');
+  });
+  
+  $('#close-btn').click(function () {
+    $('#embedModal').modal('hide');
+  });
 </script>
 
 {# Bottom: determines whether to have view or embed button
 # based on embed flag #}
 <div id="bottom">
-  <div class="container row">
-    <div class="span5 offset1"><h3>{{_('Previewing')}}: {{ file.filename }}</h3></div>
-    <div class="span3">
-      <a role="button" class="btn btn-inverse" data-toggle="modal" href="#embedModal">{{_('Embed')}}&nbsp;&nbsp<i class="icon-share icon-white"></i></a>
-      <a role="button" class="btn btn-inverse" href="{{ file.uri }}">{{_('Download')}}&nbsp;&nbsp<i class="icon-download icon-white"></i></a>
+  <div class="ui grid container">
+    <div class="row"></div>
```

Comment:
minor: is this empty row needed?

Reply:
It adds some space above the `Previewing` title. I could do it with css too.

Reply:
If space is needed, maybe the `padded` class can be added.